edx_proctoring/tests/test_views.py

```diff
@@ -414,6 +414,7 @@ def setUp(self):
         self.student_taking_exam = User()
         self.student_taking_exam.save()
 
+        set_runtime_service('credit', MockCreditService())
```

Comment:
Good question. These tests call proctoring code that assumes that there's a credit service installed. They work when run as a complete suite because other tests have installed the service and it doesn't get uninstalled. I put this in so that I can run individual suites and/or test methods.